src/CommandingOfficers/Tech.java

```diff
@@ -0,0 +1,452 @@
+package CommandingOfficers;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.PriorityQueue;
+import java.util.Random;
+import java.util.Set;
+
+import AI.AIUtils;
+import CommandingOfficers.Modifiers.CODamageModifier;
+import CommandingOfficers.Modifiers.CODefenseModifier;
+import Engine.GameScenario;
+import Engine.UnitActionFactory;
+import Engine.Utils;
+import Engine.XYCoord;
+import Engine.GameEvents.GameEventQueue;
+import Engine.GameEvents.TeleportEvent;
+import Terrain.Location;
+import Terrain.MapMaster;
+import Units.Unit;
+import Units.UnitModel;
+
+public class Tech extends Commander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Tech");
+      infoPages.add(new InfoPage(
+          "Tech is a first-rate grease monkey who like nothing better than to build new machines and set them loose.\n"));
+      infoPages.add(new InfoPage(
+          "Passive:\n" +
+          "- Tech has some of the best mechanics around, allowing her units to repair 3 HP per turn instead of just 2.\n"));
+      infoPages.add(new InfoPage(
+          TECHDROP_NAME + " (" + TECHDROP_COST + "):\n" +
+              "Deploys " + TECHDROP_NUM + " BattleMech to the front lines.\n" +
+              "All mechanical units gain " + TECHDROP_BUFF + "% attack.\n" +
```

Comment:
You're buffing both attack and defense, though.

Reply:
Read the next line?

Reply:
...good job, me.